packages/dripsy/src/css/create-themed-component.tsx

```diff
@@ -105,7 +105,6 @@ export function createThemedComponent<P, T>(
   })
 
   WrappedComponent.displayName = `Themed.${
-    // Optional chaining here prevents the Pressable component causing a crash on older versions of RNW
     Component?.displayName ?? 'NoNameComponent'
```

Comment:
did this issue get solved?

Reply:
No, the issue still persists. I'll add the comment back if you think it's important. I can also create a warning when the Pressable or another component is not found that it might be a result of an older version of RNW.